.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+install:
+  - pip install -r requirements.txt
+  - git clone https://github.com/simphony/JYU-LB.git
```

Comment:
@itziakos  and @kemattil ,  this is checking out JYU-LB  (and not simphony-jyulb).  

simphony-jyulb needs JYU-LB.  so its checked out here and built.